t04_refactored.py

```diff
@@ -80,7 +82,7 @@ def scott_adventure():
 
     :return: None
     """
-    global dead             # You'll need this to be able to modify the dead variable
```

Comment:
Changes should not have been made outside of your function. Please revert these.